spotbugs-exclude.xml

```diff
@@ -0,0 +1,5 @@
+<FindBugsFilter>
+    <Match>
+        <Bug code="EI,EI2,MS"/>
```

Comment:
EI: May expose internal representation by returning a buffer sharing non-public data (EI_EXPOSE_BUF)

EI2: May expose internal representation by incorporating reference to mutable object (EI_EXPOSE_REP2)

MS: May expose internal static state by storing a mutable object into a static field (EI_EXPOSE_STATIC_REP2)

This bugs have a lot of occurrences. Fixing them might cost a lot for no real improvement. I think we can ignore them